### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by triallax**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -15,14 +15,14 @@
     "test"
   ],
   "dependencies": {
-    "purescript-console": "^4.1.0",
-    "purescript-foldable-traversable": "^4.1.0",
-    "purescript-distributive": "^4.0.0",
-    "purescript-quickcheck": "^6.0.0"
```
Comment (triallax) [inline bower.json:21]: I moved this to `devDependencies` since it is only used in test code. If you'd like to make this in another PR, I don't mind.

File: test/Main.purs [hunk 2]
```diff
@@ -6,8 +6,8 @@ import Type.Proxy (Proxy(..), Proxy2(..))
 
 import Data.Array (cons, snoc, fromFoldable)
 import Data.Pair (Pair(..), (~), fst, snd, swap, uncurry)
-import Data.Foldable (foldMap, foldr, foldl)
-import Data.Traversable (sum, product, sequence)
+import Data.Foldable (foldMap, foldr, foldl, product, sum)
+import Data.Traversable (sequence)
```
Comment (triallax) [inline test/Main.purs:10]: See https://github.com/purescript/purescript-foldable-traversable/issues/137.

**Review by sharkdp — APPROVED** [review]
Thank you!

**Comment by triallax** [comment]
Just noticed #2. _facepalm_

Let me open another PR to fix this.
